### PR TITLE
Add RGB example using Mesh

### PR DESCRIPTION
**Pull request by johnpmayer**
Adding a trivial example to get familiar with the new APIs.

Probably not quite ready, would wait for a new release of blinds (alpha9 or something?)

### CODE REVIEW

**Comment by johnpmayer** [comment]
Is there a way to run the checks locally (at least for my platform?)

File: examples/03_rgb_triangle.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            size: Vector::new(800.0, 600.0).into(),
+            title: "RGB Triangle Example",
+            ..Settings::default()
+        },
+        app,
+    );
+}
+
+async fn app(window: Window, mut gfx: Graphics, mut events: EventStream) -> Result<()> {
+    // Clear the screen to a blank, white color
```
Comment (johnpmayer) [inline examples/03_rgb_triangle.rs]: well it's a black color now, update the comment

File: examples/03_rgb_triangle.rs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            pos: Vector::new(600, 400),
+            uv: None,
+            color: Color::BLUE,
+        };
+        vec![top, left, right]
+    };
+
+    let elements = vec![Element::Triangle([0, 1, 2])];
+
+    let image = None;
```
Comment (johnpmayer) [inline examples/03_rgb_triangle.rs]: eh, I'm not convinced this is worthwhile, just inline it.

File: examples/03_rgb_triangle.rs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    let mesh = Mesh {
+        vertices,
+        elements,
+        image,
+    };
+
+    gfx.draw_mesh(&mesh);
```
Comment (johnpmayer) [inline examples/03_rgb_triangle.rs]: this should be more verbosely commented, overall

Comment (johnpmayer) [reply examples/03_rgb_triangle.rs]: done

File: examples/03_rgb_triangle.rs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            color: Color::GREEN,
+        };
+        let right = Vertex {
+            pos: Vector::new(600, 400),
+            uv: None,
+            color: Color::BLUE,
+        };
+        vec![top, left, right]
+    };
+    // A triangle is simply a pointer to indices of the vertices
```
Comment (johnpmayer) [inline examples/03_rgb_triangle.rs]: is "pointer" misleading?

**Comment by ryanisaacg** [comment]
This looks really good! I'll merge it now, and we can move it around later if necessary (I'm not sure `Mesh` will remain the 3rd example)

**Pull request merged**